eng/pipelines/templates/stages/archetype-python-release.yml

```diff
@@ -271,3 +271,34 @@ stages:
               twine upload --repository $(DevFeedName) --config-file $(PYPIRC_PATH) $(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{artifact.name}}/*-*a*.zip
               echo "Uploaded sdist to devops feed $(DevFeedName)"
             displayName: 'Publish ${{artifact.name}} alpha package'
+
+    - job: PublishDocsToNightlyBranch
+      condition: or(eq(variables['SetDevVersion'], 'true'), and(eq(variables['SetDevVersion'], ''), eq(variables['Build.Reason'],'Schedule'), eq(variables['System.TeamProject'], 'internal')))
```

Comment:
Should we have the SetDevVersion="" condition here?

Reply:
It triggered two ways, if the dev version is true, or it is nightly built. It does not need the step once it is nightly build pipeline.

Reply:
See https://github.com/Azure/azure-sdk-tools/issues/2037 we should remove the SetDevVersion="" condition.